test_scripts/WebEngine/commonWebEngine.lua

```diff
@@ -99,6 +102,26 @@ function common.validation(actualData, expectedData, pMessage)
   return true
 end
 
+function common.updatePreloadedPT(pAppId, pAppHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  preloadedTable.policy_table.app_policies[appId] = common.cloneTable(preloadedTable.policy_table.app_policies.default)
+  preloadedTable.policy_table.app_policies[appId].AppHMIType = { pAppHMIType }
+  preloadedTable.policy_table.app_policies[appId].enabled = true
+  preloadedTable.policy_table.functional_groupings["DataConsent-2"].rpcs = common.null
```

Comment:
@ydementieiev It is to add only `enabled` flag is not enough to set webengine application in policy table.
```suggestion
  local appPermissions = common.cloneTable(preloadedTable.policy_table.app_policies.default)
  appPermissions.AppHMIType = { pAppHMIType }
  appPermissions.enabled = true
  appPermissions.transportType = "WS",
  appPermissions.hybridAppPreference = "CLOUD"
  preloadedTable.policy_table.app_policies[appId] = appPermissions
  preloadedTable.policy_table.functional_groupings["DataConsent-2"].rpcs = common.null
```

Reply:
@aderiabin thank you , done